content/2-5.md

```diff
@@ -137,114 +146,102 @@ Explain how form instructions allow people to understand the purpose of form con
+* Show examples of required form controls using the HTML attribute `required` and corresponding text added in the label. Explain why it is necessary to provide this information in multiple ways so that it is available to all users, rather than to rely on color alone or on the HTML semantics alone.
 
 #### Ideas to Assess Knowledge for Topic
 
 Optional ideas to support assessment:
 
-* Practical &mdash; Students are presented with overall instructions for a set of form controls such as the current and total number of steps of the form and are asked to code them appropriately. Assess students' knowledge about how to code overall instructions.
+* Practical &mdash; Students are presented with overall instructions for a set of form controls such as the current and total number of steps of the form and are asked to code them appropriately. Assess students' knowledge of how to code overall instructions.
 * Practical &mdash; Students are presented with in-line instructions for form controls such as examples of how data should be entered and are asked to code them appropriately. Assess students' knowledge of the different techniques to code in-line instructions.
+* Practical &mdash; Students are presented with a set of form controls and are asked to code them so that they are required. Assess how students make use of the HTML attribute `required` and corresponding text in the label of the form control.
 
 {% include excol.html type="end" %}
 
 {% include excol.html type="start" %}
 
-### Topic: Time Limits
+### Topic: Notifications
 
 {% include excol.html type="middle" %}
 
-Explain how to code mechanisms to stop, adjust, or extend time limits as well as general instructions for time limits.
+Explain the use of HTML and WAI-ARIA to provide notifications about errors, successes, and other feedback resulting from interaction by the user. This includes feedback at the end of the process, such as submitting a form, and within the process, such as from validating input as the user types.
 
 #### Learning Outcomes for Topic
 
 Students should be able to:
 
-* code mechanisms to stop, adjust, or extend time limits of a given form
-* List requirements for designers and content authors to write overall timing-related instructions, such as conveying the time limit, and explaining the ways to stop or extend it
+* recite related requirements for content authors and designers to:
+  * provide error and success messages when sessions and transactions are completed
+  * provide clear feedback on the status of transition from one step of the process to another in multi-step forms
+  * ensure that error and success messages that appear when forms are submitted are easy to perceive and understand by users
+  * ensure that in-line error messages and other feedback that appears from user interaction with forms is clearly identifiable
+  * provide warnings about any ensuing time limits, and appropriate mechanisms to stop, adjust, or extend such time limits
```

Comment:
"provide warnings about any *imminent* time limits" not 'ensuing'

Reply:
Will change.